<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IcqOptionsWidget</class>
 <widget class="QWidget" name="IcqOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ICQ</string>
  </property>
  <widget class="QToolButton" name="btnIcqAccountsSwitch">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>135</y>
     <width>27</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch</string>
   </property>
   <property name="icon">
    <iconset resource="../../qHostMonitor.qrc">
     <normaloff>:/img/jabberSwitch.png</normaloff>:/img/jabberSwitch.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="gpPrimaryIcq">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>25</y>
     <width>340</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string> Primary ICQ account  </string>
   </property>
   <widget class="QComboBox" name="cmbPrimaryIcqUin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>31</y>
      <width>240</width>
      <height>27</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPrimaryIcqPassword">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>61</y>
      <width>240</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPrimaryIcqUin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>UIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPrimaryIcqPassword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gpBackupIcq">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>340</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string> Backup ICQ account  </string>
   </property>
   <widget class="QLineEdit" name="editBackupIcqPassword">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>61</y>
      <width>240</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBackupIcqUin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>31</y>
      <width>240</width>
      <height>27</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblBackupIcqUin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>UIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBackupIcqPassword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
